### PR TITLE
[#929] Modification to comand vela show WORKLOAD_TYPE or TRAIT

### PR DESCRIPTION
**Pull request by just-do1**

### CODE REVIEW

File: pkg/commands/show.go [hunk 1]
```diff
@@ -56,16 +56,17 @@ func NewAppShowCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Command
...
 			}
-			return startReferenceDocsSite(ctx, c, ioStreams, capabilityName)
+			return showReferenceConsole(ctx, c, ioStreams, capabilityName)
 		},
 		Annotations: map[string]string{
 			types.TagCommandType: types.TypeStart,
 		},
 	}
-	cmd.Flags().BoolVarP(&noWebSite, "no-website", "", false, "do not start web doc site")
+	// cmd.Flags().BoolVarP(&noWebSite, "no-website", "", false, "do not start web doc site")
```
Comment (wonderflow) [inline pkg/commands/show.go:68]: let's remove it

**Review by wonderflow — COMMENTED** [review]
Generally  LGTM, please also update the doc

**Comment by wonderflow** [comment]
You can add your git email to your github emails list to show yourself in commit message.

**Comment by just-do1** [comment]
OK， I will to add the email

**Comment by wonderflow** [comment]
please also update the doc https://github.com/oam-dev/kubevela/blob/master/docs/en/developers/check-ref-doc.md#the-reference-documentation-guide-of-capabilities

**Comment by just-do1** [comment]
ok, I will do it

**Comment by just-do1** [comment]
What should I do about this failing?  
The check of Dashboard/build is error

**Review by wonderflow — APPROVED** [review]
LGTM

**Pull request merged**